components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/internal/Suggestion.kt

```diff
@@ -2,34 +2,41 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-package mozilla.components.compose.browser.awesomebar
+package mozilla.components.compose.browser.awesomebar.internal
 
 import android.graphics.Bitmap
+import android.graphics.drawable.Drawable
 import androidx.compose.foundation.Image
 import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.IntrinsicSize
 import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.width
-import androidx.compose.material.ContentAlpha
-import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.rotate
 import androidx.compose.ui.graphics.ColorFilter
 import androidx.compose.ui.graphics.asImageBitmap
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import androidx.core.graphics.drawable.toBitmap
+import mozilla.components.compose.browser.awesomebar.AwesomeBarColors
+import mozilla.components.compose.browser.awesomebar.AwesomeBarOrientation
+import mozilla.components.compose.browser.awesomebar.R
 import mozilla.components.concept.awesomebar.AwesomeBar
 
 @Composable
 internal fun Suggestion(
```

Comment:
Should we be creating any previews when creating new composables? (or did I miss them)

Reply:
It's a bit tricky with `AwesomeBar.Suggestion` which have a reference to a `SuggestionProvider`, so there's a bunch of stuff to mock. Not impossible, but was too much for this PR. I have been using the sample app in the repo for testing.

Reply:
Gotcha ok. It might be worth creating some abstracted-out or dependency-free composables to make previewing easier then.

Reply:
Yeah, we could unpack the `Suggestion` at the top and pass the individual values down. This would introduce a bunch of parameters though.
Another option we have is: Once we have fully migrated to the composable, we can simplify `Suggestion`, since some of the things, like the mentioned reference, are only needed for the `BrowserAwesomeBar` implementation and then it will be easier to use previews here too :)